pipelines/batch/src/main/java/org/openmrs/analytics/ConvertResourceFn.java

```diff
@@ -76,6 +79,21 @@ public void writeResource(HapiRowDescriptor element)
             .setVersionId(element.resourceVersion())
             .setLastUpdated(simpleDateFormat.parse(element.lastUpdated()));
 
+    if (element.getTags() != null) {
+      List<Coding> tags = new ArrayList<>();
+      for (ResourceTag resourceTag : element.getTags()) {
+        // The column tag_type of value 0 means it's of type TAG, 1 for PROFILE and 2 for SYSTEM.
+        if (resourceTag.getTagType() == 1) {
+          CanonicalType canonicalType = new CanonicalType();
+          canonicalType.setValue(resourceTag.getCoding().getCode());
+          meta.setProfile(List.of(canonicalType));
```

Comment:
Is it ever possible that there maybe multiple profiles?

Reply:
I did not observe multiple entries in the db, although I will make the changes, thanks!